src/main/java/uk/gov/hmcts/reform/sscs/model/servicehearingvalues/PartyDetails.java

```diff
@@ -25,8 +25,7 @@
     private String partyID;
     @EnumPattern(enumClass = PartyType.class, fieldName = "partyType")
```

Comment:
You've made these changes in serviceHearingValues which is for the ExUI request. You want to make the changes in the single/hearing package which is the HMC one

Reply:
Swagger gets confused because of the object names, we're changing them around but for now you can remove the @PostMapping from ServiceHearingValuesController to stop Swagger getting confused.